tests/javascript/testrunnerPhantom.js

```diff
@@ -23,7 +23,23 @@
 
 var fs  = require("fs");
 var system = require("system");
-var url = system.args[1] || 'http://localhost/tests/javascript/';
+var baseUrl = system.args[1] || 'http://localhost/tests/javascript/';
+
+function getPluginArg() {
+  for (var i = 0; i < system.args.length; ++i) {
+    if (/--plugin=(.*?)/.test(system.args[i])) {
+      return system.args[i].split('=', 2)[1];
+    }
+  }
+  return null;
+}
```

Comment:
Is it possible to use the same approach as in the nodeRunner?

```suggestion
function getPluginArg() {
  const pluginArg = system.args.find((arg) => /--plugin=(.*?)/.test(arg));
  return pluginArg ? pluginArg.split('=', 2)[1] : null;
}
```

Reply:
phantomjs does not support modern ecmascript syntax and I'm not sure if it has ES5 array functions.

Reply:
Thanks, that's definitely a good reason to keep it the old way.